plugins/ci/pkg/models/models.go

```diff
@@ -28,10 +32,11 @@ type ReportInfo struct {
 
 // Configuration is a struct representing the config options for Insights CI/CD
 type Configuration struct {
-	Images    imageConfig    `yaml:"images"`
-	Manifests ManifestConfig `yaml:"manifests"`
-	Options   optionConfig   `yaml:"options"`
-	Reports   reportsConfig  `yaml:"reports"`
+	RepoBasePath string         `yaml:"-"`
```

Comment:
this variable can be
- `.` if running normally
- `/app/repository/{repoName}` if running in cloned repo context